test/unit/annotation_layer_spec.js

```diff
@@ -499,13 +500,18 @@ describe('Annotation layer', function() {
       expect(textWidgetAnnotation.data.maxLen).toEqual(null);
       expect(textWidgetAnnotation.data.readOnly).toEqual(false);
       expect(textWidgetAnnotation.data.multiLine).toEqual(false);
+      expect(textWidgetAnnotation.data.comb).toEqual(false);
     });
 
     it('should set valid text alignment, maximum length and flags',
         function() {
+      var flags = 0;
+      flags |= 1 << (AnnotationFieldFlag.READONLY - 1);
+      flags |= 1 << (AnnotationFieldFlag.MULTILINE - 1);
```

Comment:
As far as I'm concerted, the current way that the `AnnotationFieldFlag` are defined makes the unit-tests unnecessarily convoluted and difficult to understand.

What would be *a lot* clearer is if you changed `AnnotationFieldFlag` to the same form as `AnnotationFlag`, since that means you could change the `hasFieldFlags` method in `src/core/annotation.js` to just:
```javascript
return !!(this.data.fieldFlags & flag);`
```
and specifying the flags in the unit-tests would be so much more straightforward:
```javascript
var flags = AnnotationFieldFlag.READONLY + AnnotationFieldFlag.MULTILINE;
```